OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Different editors fight over the formatting of the csproj files.  Please unstage/discard all except the file addition.